core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1066,6 +1066,31 @@ public void listStatus(AlluxioURI path, ListStatusContext context,
     return fileInfos;
   }
 
+  /**
+   * {@link DefaultFileSystemMaster} has two private methods associated with sync metadata:
+   * 1. {@link DefaultFileSystemMaster#loadMetadataIfNotExist} sync metadata in load-only mode,
```

Comment:
what is load-only mode?

Reply:
load-only mode won't update metadata which is not consistency with UFS, this mode will load metadata which not in alluxio inode tree but exists in UFS.